fdbrpc/Locality.cpp

```diff
@@ -242,10 +253,22 @@ ProcessClass::Fitness ProcessClass::machineClassFitness(ClusterRole role) const
 		case ProcessClass::CoordinatorClass:
 		case ProcessClass::TesterClass:
 		case ProcessClass::StorageCacheClass:
+		case ProcessClass::BlobWorkerClass:
 			return ProcessClass::NeverAssign;
 		default:
 			return ProcessClass::WorstFit;
 		}
+	case ProcessClass::BlobWorker:
+		switch (_class) {
+		case ProcessClass::BlobWorker:
+			return ProcessClass::BestFit;
+		// TODO: do we want this? If there's an unset process available,
```

Comment:
We can just leave it with no blob workers for now.
We should probably eventually implement a "critical recruitment" type thing like DD does for SS where it is ok with worse fit matches for recruitment, including unset, but we can do that as a separate change

Reply:
I like that as well! thanks for the suggestion :)